specs/windows/physical_disk_perf.table

```diff
@@ -0,0 +1,17 @@
+table_name("physical_disk_perf")
+description("Provides provides raw data from performance counters that monitor hard or fixed disk drives on the system.")
+schema([
+    Column("name", TEXT, "Name of the physical disk"),
+    Column("avg_disk_bytes_per_read", BIGINT, "Average number of bytes transferred from the disk during read operations"),
```

Comment:
These names are wacky but if you feel there's value in mimicking the names from the WMI table then keep them equivalent.

Reply:
I agree they're wacky. Not sure I have a solution that outweighs keeping equivalence with the WMI table spec however. I'm definitely open to suggestions for them, but for folks comfortable with Perfomn it might be better to keep them the same.

This being the first (of hopefully many) Perfmon table, if we'd like to experiment with differing table schema names it might be a good time to pioneer things. I'll throw out to the #windows channel and see if any one has suggestions/thoughts on the matter.